### PR TITLE
Add twig/string-extra extension

### PR DESCRIPTION
**Pull request by nicolas-grekas**
Last step before deprecating https://github.com/twigphp/Twig-extensions entirely.

See https://github.com/twigphp/Twig/blob/2.x/doc/filters/u.rst for doc.

### CODE REVIEW

File: extra/string-extra/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+String Extension
+================
+
+This package is a Twig extension that provides integration with the Symfony String component.
+
+It provides a single [`u`][1] filter that wraps a text in a `UnicodeString` object to give access to [methods of the class](https://symfony.com/doc/current/components/string.html).
+
+`{{ 'Symfony String + Twig = <3'|u.wordwrap(5) }}`
```
Comment (fabpot) [inline extra/string-extra/README.md]: Can we add some more relevant examples?

Comment (fabpot) [reply extra/string-extra/README.md]: This should be moved to the docs anyway.

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

File: CHANGELOG [hunk 2]
```diff
@@ -1,14 +1,14 @@
 * 2.12.1 (2019-XX-XX)
 
- * n/a
+ * added the String extension in the "extra" repositories: "u" filter
```
Comment (lyrixx) [inline CHANGELOG:3]: It should have been better to add it in 2.13.0

**Pull request merged**